run/test/specs/group_tests_add_contact.spec.ts

```diff
@@ -49,16 +49,7 @@ async function addContactToGroup(platform: SupportedPlatformsType) {
   // Click more options
   await device1.clickOnByAccessibilityID("More options");
   // Select edit group
-  await runOnlyOnIOS(platform, () =>
-    device1.clickOnByAccessibilityID("Edit group")
-  );
-  await sleepFor(1000);
```

Comment:
you've also removed the sleepFor, by mistake?

Reply:
yes indeed